src/gameplay_v2/roles/assassin.ts

```diff
@@ -0,0 +1,13 @@
+import Entity from './entity';
+import { Alliance } from '../gameTypes';
+import { assassinateC } from './components/assassinate';
+
+export default class Assassin extends Entity {
+  description: string;
+
+  constructor() {
+    super(Alliance.Resistance);
+    this.description = 'If the resistance win 3 missions, the Assassin can shoot one person for Merlin, or two people for Tristan and Isolde. If they are correct, the spies win!';
+    this.components.push(new assassinateC());  
```

Comment:
I remember bringing this up a while ago, but we might be better off using a hashmap instead of an array so it's easier to access stuff. Have you thought about this?

Reply:
Aaahh, I thought about that but was unsure on how it will go but I'll try on it.